Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -382,3 +366,102 @@ void GeneralPane::GenerateNewIdentity()
   message_box.exec();
 }
 #endif
+
+void GeneralPane::AddDescriptions()
+{
+  static constexpr char TR_DUALCORE_DESCRIPTION[] = QT_TR_NOOP(
+      "Runs the emulated CPU and GPU on separate threads. Increases Dolphin's frame rate, but can "
+      "result in various glitches and crashes."
+      "<br><br>This setting cannot be changed while emulation is active."
+      "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_CHEATS_DESCRIPTION[] = QT_TR_NOOP(
+      "Enables the use of AR and Gecko cheat codes which can be used to modify games' behavior. "
+      "These codes can be configured with the Cheats Manager in the Tools menu."
+      "<br><br>This setting cannot be changed while emulation is active."
+      "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_OVERRIDE_REGION_SETTINGS_DESCRIPTION[] =
+      QT_TR_NOOP("Uses the language or region set by the GameCube or Wii instead of the one "
+                 "specified by the game. Fixes various crashes and bugs."
+                 "<br><br>This setting cannot be changed while emulation is active."
+                 "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_AUTO_DISC_CHANGE_DESCRIPTION[] = QT_TR_NOOP(
+      "Automatically changes the game disc when requested by games with 2 discs. This feature "
+      "requires the game to be launched in one of the following ways:"
+      "<br>- From the game list, with both discs being present in the game list."
+      "<br>- With File > Open or the command line interface, with the paths to both discs being "
+      "provided."
+      "<br>- By launching an M3U file with File > Open or the command line interface."
+      "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+#ifdef USE_DISCORD_PRESENCE
+  static constexpr char TR_DISCORD_PRESENCE_DESCRIPTION[] =
+      QT_TR_NOOP("Shows which game is active in your Discord status and the duration of your "
+                 "current play session."
+                 "<br><br>This setting cannot be changed while emulation is active."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+#endif
+  static constexpr char TR_SPEEDLIMIT_DESCRIPTION[] =
+      QT_TR_NOOP("Sets the maximum speed at which games will run. If your computer is not fast "
+                 "enough the actual speed may be slower."
+                 "<br><br><dolphin_emphasis>If unsure, select 100%.</dolphin_emphasis>");
+  static constexpr char TR_UPDATE_TRACK_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects which update track Dolphin uses when checking for updates at startup. If a new "
+      "update is available Dolphin will show a list of changes made since your current version and "
+      "ask you if you want to update."
```

Comment:
```suggestion
      "Selects which update track Dolphin uses when checking for updates at startup. If a new "
      "update is available, Dolphin will show a list of changes made since your current version and "
      "ask you if you want to update."
```